raddb/mods-available/ldap

```diff
@@ -135,231 +191,287 @@ ldap {
 #		&reply:Tunnel-Medium-Type	:= 'radiusTunnelMediumType'
 #		&reply:Tunnel-Private-Group-ID	:= 'radiusTunnelPrivategroupId'
 
-		#  Where only a list is specified as the RADIUS attribute,
+		#  NOTE: Where only a list is specified as the RADIUS attribute,
 		#  the value of the LDAP attribute is parsed as a valuepair
 		#  in the same format as the 'valuepair_attribute' (above).
 		&control:			+= 'radiusControlAttribute'
 		&request:			+= 'radiusRequestAttribute'
 		&reply:				+= 'radiusReplyAttribute'
 	}
 
-	#  Set to yes if you have eDirectory and want to use the universal
+	#
+	#  edir:: Set to `yes` if you have eDirectory and want to use the universal
 	#  password mechanism.
+	#
 #	edir = no
 
-	#  Set to yes if you want to bind as the user after retrieving the
-	#  Cleartext-Password. This will consume the login grace, and
-	#  verify user authorization.
-#	edir_autz = no
-
-	#  Note: set_auth_type was removed in v3.x.x
-	#  Equivalent functionality can be achieved by adding the following
-	#  stanza to the recv Access-Request {} section of your virtual server.
 	#
-	#    ldap
-	#    if ((ok || updated) && User-Password) {
-	#        update {
-	#            control:Auth-Type := ldap
-	#        }
-	#    }
```

Comment:
This should not be removed.  People still need to know how to set Auth-Type := ldap.  It should go in an unlang code block.